test/controllers/concerns/authentication_concern_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class AuthenticationConcernTest < ActionController::TestCase
+  setup do
+    @controller = SessionsController.new
+  end
+
+  def test_unauthenticate_user
+    # should disconnect all the connections established for current users
+    user = User.create(name: 'Sticky Fingaz')
+
+    post :create, params: { user_id: user.id }
+
+    ActionCable.server.expects(:disconnect).with(has_entries(current_user: user))
+
+    delete :destroy
+  end
+end
```

Comment:
Agree with you, maybe better in future write capybara test, It is exactly the same problem as in issue (login, link to comments, logout, post new comment on opened tab etc.)

Ok, I just nix it now